### PR TITLE
Update UI configuration

### PR DESCRIPTION
**Pull request by kit-klein**
Add details on additional Konnected panel behavior settings in the UI

## Proposed change

This change adds doc details related non-zone specific UI settings for Konnected Alarm Panels.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/33481
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/konnected.markdown [hunk 1]
```diff
@@ -93,6 +93,16 @@ The settings UI starts by having you configure the general behavior of each zone
 
 **Configure additional states for this zone:** Selecting "No" will complete configuration for the zone and proceed to options for the next zone. Select "Yes" if you need to create additional output states for this zone.  
 
+#### Using Settings UI to Configure Additional Panel Behavior
+
+Once all zones are configured you'll be presented with configuration for additional panel behaviors.
+
+**Blink panel LED on when sending state change:** The desired LED behavior for the panel
+
+**Override default Home Assistant API host panel URL:** The Konnected Alarm Panel post sensor states back to the Home Assistant API.  If this value is unchecked the panel will default postbacks using `base_url` in the `http` component. If you've set `base_url` to an external hostname, then you'll want to check this field and set the **Override API host url** to your _local_ IP address and port (e.g., `http://192.168.1.101:8123`).
```
Comment (frenck) [inline source/_integrations/konnected.markdown]:
```suggestion
**Override default Home Assistant API host panel URL:** The Konnected Alarm Panel post sensor states back to the Home Assistant API. If this value is unchecked the panel will default postbacks using `base_url` in the `http` component. If you've set `base_url` to an external hostname, then you'll want to check this field and set the **Override API host URL** to your _local_ IP address and port (e.g., `http://192.168.1.101:8123`).
```

File: source/_integrations/konnected.markdown [hunk 2]
```diff
@@ -93,6 +93,16 @@ The settings UI starts by having you configure the general behavior of each zone
 
 **Configure additional states for this zone:** Selecting "No" will complete configuration for the zone and proceed to options for the next zone. Select "Yes" if you need to create additional output states for this zone.  
 
+#### Using Settings UI to Configure Additional Panel Behavior
+
+Once all zones are configured you'll be presented with configuration for additional panel behaviors.
+
+**Blink panel LED on when sending state change:** The desired LED behavior for the panel
```
Comment (frenck) [inline source/_integrations/konnected.markdown]:
```suggestion
**Blink panel LED on when sending state change:** The desired LED behavior for the panel.
```

File: source/_integrations/konnected.markdown [hunk 3]
```diff
@@ -93,6 +93,16 @@ The settings UI starts by having you configure the general behavior of each zone
 
 **Configure additional states for this zone:** Selecting "No" will complete configuration for the zone and proceed to options for the next zone. Select "Yes" if you need to create additional output states for this zone.  
 
+#### Using Settings UI to Configure Additional Panel Behavior
+
+Once all zones are configured you'll be presented with configuration for additional panel behaviors.
```
Comment (frenck) [inline source/_integrations/konnected.markdown]:
```suggestion
Once all zones are configured you'll be presented with the configuration for additional panel behaviors.
```

File: source/_integrations/konnected.markdown [hunk 4]
```diff
@@ -93,6 +93,16 @@ The settings UI starts by having you configure the general behavior of each zone
...
+
+**Blink panel LED on when sending state change:** The desired LED behavior for the panel
+
+**Override default Home Assistant API host panel URL:** The Konnected Alarm Panel post sensor states back to the Home Assistant API.  If this value is unchecked the panel will default postbacks using `base_url` in the `http` component. If you've set `base_url` to an external hostname, then you'll want to check this field and set the **Override API host url** to your _local_ IP address and port (e.g., `http://192.168.1.101:8123`).
+
+**Override API host URL (optional):** The host info to use if you checked **Override default Home Assistant API host panel URL** in the step above.  This is ignored if **Override default Home Assistant API host panel URL** is unchecked.
```
Comment (frenck) [inline source/_integrations/konnected.markdown]:
```suggestion
**Override API host URL (optional):** The host info to use if you checked **Override default Home Assistant API host panel URL** in the step above. This is ignored if **Override default Home Assistant API host panel URL** is unchecked.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks, some small notes @kit-klein! 👍

File: source/_integrations/konnected.markdown (hunk 4 above)
Comment (kit-klein) [reply source/_integrations/konnected.markdown]: Sorry - old habit throwing that extra space in there 🤕. I know it's against guidelines...

File: source/_integrations/konnected.markdown (hunk 1 above)
Comment (kit-klein) [reply source/_integrations/konnected.markdown]: This should be fixed by an early commit I must have done when you were reviewing.

File: source/_integrations/konnected.markdown (hunk 4 above)
Comment (frenck) [reply source/_integrations/konnected.markdown]: Hehe don't worry, we'll catch them 😉

**Review by frenck — APPROVED** [review]
Thanks, @kit-klein! 👍

**Pull request merged**